### PR TITLE
Tempdir param

### PR DESCRIPTION
**Pull request by cynicer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,22 +36,13 @@ Background: If your app is Bitcode-enabled, then the dSYMs that are generated du
...
 
-To fully automate the following five-step workflow, add the snippets below to the respective files and fill in the placeholders. Uploading the app the App Store Connect is a necessary prerequisite and either handled manually or by fastlane directly:
+To fully automate the following five-step workflow, add the snippets below to the respective files and fill in the placeholders. Uploading the app to App Store Connect is a necessary prerequisite and either handled manually or by fastlane directly:
 
 1. Wait until the build is processed
 2. Download the resulting dSYM files
 3. Process dSYM files into the format that Dynatrace requires
 4. Upload processed dSYM files to Dynatrace
 
-
```
Comment (cynicer) [inline README.md:48]: unused

File: README.md [hunk 2]
```diff
@@ -114,13 +103,10 @@ dynatrace_process_symbols(
...
 | symbolsfile                  | Path to the dSYM file to be processed. If downloadDsyms is set, this is only a fallback.                                                                              |                |
 | server                       | The API endpoint for the Dynatrace environment (e.g. `https://environmentID.live.dynatrace.com` or `https://dynatrace-managed.com/e/environmentID`).                  |                |
-| cleanBuildArtifacts | Clean build artifacts after processing.                                                                                         | `true`         |
+| cleanBuildArtifacts          | Clean build artifacts after processing.                                                                                         | `true`         |
+| tempdir                      | (OPTIONAL) Custom temporary directory for the DTXDssClient. **The plugin does not take care of cleaning this directory.**                                                                                         | `true`         |
```
Comment (Landsi) [inline README.md:107]: default value 'true' doesn't seem like the best fit :) -> nil/null/none?

**Review by Landsi — APPROVED** [review]
lgtm, besides the small readme issue

File: README.md (hunk 2 above)
Comment (harp79) [reply README.md:107]: or default "not set" - uses default tmp dir

**Review by harp79 — APPROVED** [review]

**Review by Landsi — APPROVED** [review]

**Pull request merged**